u2f-gae-demo/src/com/google/u2f/gaedemo/servlets/BeginEnrollServlet.java

```diff
@@ -51,8 +51,8 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
       throw new ServletException("couldn't get registration request", e);
     }
 
-	  JsonObject result = new JsonObject();
-    result.addProperty("appId", registrationRequest.getAppId());
+    JsonObject result = new JsonObject();
```

Comment:
Can you fix the indentation?  It might look nice in some editor, but there is clearly something wrong here.

```JsonObject result = new JsonObject();```

should have the same indent level as ```try```.

Also, things are messed up below as well.  Basically, this whole file needs to be formatted and tabs expanded to spaces.